### PR TITLE
Changed emojis 

### PR DESCRIPTION
**Pull request by prynacshrma**
Changed emojis as per global slack emojis available

### CODE REVIEW

File: Parsers/weekMoji.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*
+emoji: saturday-chill
```
Comment (wiz0floyd) [inline Parsers/weekMoji.js:2]: This trigger is not a valid emoji. Thank you for making the other changes. Maybe the calendar emoji would be a good trigger?

**Review by wiz0floyd — CHANGES_REQUESTED** [review]

File: Parsers/weekMoji.js (hunk 1 above)
Comment (prynacshrma) [reply Parsers/weekMoji.js]: Made the changes in header. Thanks

**Review by Lacah — APPROVED** [review]

**Comment by Lacah** [comment]
LGTM

File: Parsers/weekMoji.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/*
+emoji: calendar
```
[the remark this answers is not in the document]
Comment (prynacshrma) [reply Parsers/weekMoji.js:2]: Removed trailing spaces

**Pull request merged**